app/index.ts

```diff
@@ -0,0 +1,146 @@
+import express, { Express, Request, Response } from "express";
+import { fromHexAddress, getNetwork } from "./utils";
+import { Context, PrivkeyAccount, Client, RPCClient } from "firovm-sdk";
+import tokens from "./tokens.json";
+import "dotenv/config";
+
+const app: Express = express();
+const port = Number(process.env.PORT) || 8123;
+const bind = process.env.BIND || "0.0.0.0";
+const network = process.env.NETWORK || "Regtest"; // ["Mainnet", "Testnet", "Regtest"]
```

Comment:
Configuration from environment should be all upper cases

Reply:
ok kub